### PR TITLE
[11.0][MIG] stock_cycle_count

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11.

### CODE REVIEW

File: stock_cycle_count/models/stock_cycle_count.py [hunk 1]
```diff
@@ -11,6 +10,7 @@ class StockCycleCount(models.Model):
     _name = 'stock.cycle.count'
     _description = "Stock Cycle Counts"
     _inherit = 'mail.thread'
+    _order = "id desc"
```
Comment (rousseldenis) [inline stock_cycle_count/models/stock_cycle_count.py:13]: @lreficent Is this change can break behaviours with existing data (e.g. : when migrating)?

**Review by rousseldenis — APPROVED** [review]
Code review.

File: stock_cycle_count/models/stock_cycle_count.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply stock_cycle_count/models/stock_cycle_count.py:13]: @rousseldenis How? 

FYI I'm currently working on a migration that includes this module, if it generates any issue I will probably see it soon.

Comment (rousseldenis) [reply stock_cycle_count/models/stock_cycle_count.py:13]: I don't know the flow exactly but sometimes changing something small as the order can lead to weird behaviours usually when manipulating multiple recordsets (as it changes recordset ordering).

Comment (LoisRForgeFlow) [reply stock_cycle_count/models/stock_cycle_count.py:13]: interesting, I will bear it in mind and try to see if anything strange happen.

**Comment by rousseldenis** [comment]
@jbeficent @lreficent What's the status of this ?

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I think it is just fine, no problem until now.

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**